stan/math/prim/fun/grad_2F1.hpp

```diff
@@ -86,13 +86,16 @@ void grad_2F1(T& g_a1, T& g_b1, const T& a1, const T& a2, const T& b1,
     log_g_old[1] = log_t_new + log(fabs(term));
     log_g_old_sign[1] = term >= 0.0 ? log_t_new_sign : -log_t_new_sign;
 
+    T g_a1_prev = g_a1;
```

Comment:
@bbbales2 thanks for prodding me along, shouldn't we be storing these as just doubles? Is there any performance penalty involved in storing the type?

Reply:
Yeah I guess since we're just looking at the values we could do `value_of_rec`. Will push that up in a sec.